Source/WebKit/UIProcess/win/PageClientImpl.cpp

```diff
@@ -42,6 +45,11 @@
 namespace WebKit {
 using namespace WebCore;
 
+// When you call GetOpenFileName, if the size of the buffer is too small,
+// MSDN says that the first two bytes of the buffer contain the required size for the file selection, in bytes or characters
+// So we can assume the required size can't be more than the maximum value for a short.
+static const size_t maxFilePathsListSize = USHRT_MAX;
```

Comment:
change `const` to `constexpr`, and move this into `handleRunOpenPanel`.

Reply:
Fixed at https://github.com/WebKit/WebKit/commit/70ce9358458f9401040946ef3392f2ae55d6fa3e